index.js

```diff
@@ -5,5 +5,8 @@
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+	content = JSON.stringify(content)
+		.replace(/\u2028/g, '\\u2028')
+		.replace(/\u2029/g, '\\u2029');
+	return `module.exports = ${content}`;
```

Comment:
```js
return 'module.exports = ' + content;
```

Reply:
I was following the example of `json-loader` that @sokra pointed me to, where it's using template literals.

Reply:
@bmeurer - Just skip the template for now. As soon as this gets pulled into the `webpack-defaults` branch it will be converted to a template anyway.